lib/matplotlib/cbook/__init__.py

```diff
@@ -2254,9 +2267,13 @@ def _check_in_list(_values, **kwargs):
     values = _values
     for k, v in kwargs.items():
         if v not in values:
-            raise ValueError(
-                "{!r} is not a valid value for {}; supported values are {}"
-                .format(v, k, ', '.join(map(repr, values))))
+            if _print_supported_values:
+                raise ValueError(
+                    "{!r} is not a valid value for {}; supported values are {}"
```

Comment:
I'd use f-strings for the error messages; and possibly even `key, val` instead of `k, v` to make it more readable.

Reply:
Thanks for the suggestion! I just pushed the changes.